### PR TITLE
blog post for GCB 2023

### PR DESCRIPTION
**Pull request by EngyNasr**
Blog post for GCB 2023 conference

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -86,9 +86,6 @@
     "sass-loader": "^10.1.1",
     "start-server-and-test": "^1.14.0"
   },
-  "resolutions": {
```
Comment (bgruening) [inline package.json:89]: please remove changes to package.jso and yarn.lock

Comment (EngyNasr) [reply package.json]: Done :)

File: content/news/2023-09-18-gcb-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+tags: [conference, talk, bioinformatics, poster, pathogen detection]
+authors: engynasr
+supporters:
+- DECHEMA
+- ELIXIR
+components: true
+subsites: [all-eu]
+---
+
+From the 11th to the 14th of Semptember, Engy Nasr (I) and Paul Zierep from the Freiburg Galaxy team travelled to Hamburg to join and present our posters about the 'Accessible and scalable Galaxy pipelines for fast and easy (foodborne) Pathogen detection and tracking' and 'Microbiome data analysis using the Galaxy platform' at the [**German Conference on Bioinformatics (GCB) 2023**](https://gcb2023.de/).
```
Comment (paulzierep) [inline content/news/2023-09-18-gcb-2023/index.md:16]:
```suggestion
From the 11th to the 14th of Semptember, Engy Nasr (I) and Paul Zierep from the Freiburg Galaxy team travelled to Hamburg to join and present our posters about 'Accessible and scalable Galaxy pipelines for fast and easy (foodborne) Pathogen detection and tracking' and 'Microbiome data analysis using the Galaxy platform' at the [**German Conference on Bioinformatics (GCB) 2023**](https://gcb2023.de/).
```

File: content/news/2023-09-18-gcb-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+On the first day; there were multiple worshops, 7 of them were in the morning and 5 of them were in the afternoon, participants had the chance to choose one of them to attend.
+
+On the second day; we had two keynotes and two talks sessions. At the end of the second talks session, we had the poster session, were 140 posters were presented. Engy's poster, the poster presented by me, was about my project **Foodborne Pathogen Dectection using direct Nanopore** and how is it important to implement it within Galaxy. I have presented my work and showed the Pathogen Detection workflow created in Galaxy along with its [**training material**](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html) and results.
+
+<p align="center">
+<img src="engy_poster.png" width="500" height="500">
+</p>
+
+Paul has also presented his poster about **Microbiome data analysis using the Galaxy platform** with an example of our current project pAllori, where we create Galaxy workflows to perform multi mircobiome data analysis applying machine learning algorthims to identify patients with Helicobacter pylori.
+
```
Comment (paulzierep) [inline content/news/2023-09-18-gcb-2023/index.md:37]: During the poster sessions, we also had the chance to meet Anne Hartebrodt and learn about **Time shifting computation to reduce the carbon emissions of software**. This technique is particular interesting for the galaxy project, in order to improve our environmental footprint. Thanks to the contact we made during GCB, Anne will preset her work during the European galaxy days [**EGD**](https://galaxyproject.org/events/2023-10-egd/egd/).

File: content/news/2023-09-18-gcb-2023/index.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+<p align="center">
+<img src="paul_poster.png" width="500" height="500">
+</p>
+
+On the third and last day; we had more keynotes and very interesting other talks sessions, as shown in their [programme](https://gcb2023.de/Programme/_/Programm_GCB_2023%20-%2007.09.2023.pdf). Before closing the conference they annonced the winning poster, and we made it again with the Foodborne project, we won the best poster award, where I got a free conference fees for GCB 2024 in Bielefeld.
```
Comment (paulzierep) [inline content/news/2023-09-18-gcb-2023/index.md:42]:
```suggestion
On the third and last day; we had more keynotes and very interesting other talks sessions, as shown in the [programme](https://gcb2023.de/Programme/_/Programm_GCB_2023%20-%2007.09.2023.pdf). Before closing the conference they annonced the winning poster, and we made it again with the Foodborne project, we won the best poster award, where I got a free conference fee for GCB 2024 in Bielefeld.
```

**Review by paulzierep — COMMENTED** [review]
small changes

File: content/news/2023-09-18-gcb-2023/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (EngyNasr) [reply content/news/2023-09-18-gcb-2023/index.md:16]: It was a comment I got before in an older blog post, as the blog post has to have a speaker, so I relate the reader now, so instead I use Engy Nasr again, I will just say my, mine, me, etc. So i introduced this now with Engy Nasr (I)

**Comment by bgruening** [comment]
It would be nice if we could reduce the height/width of the images a bit. They are really large.

**Comment by bgruening** [comment]
Thanks @EngyNasr!

**Pull request merged**